.github/workflows/build-python-ai-scores.yaml

```diff
@@ -50,7 +53,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: 3.8
+          python-version: ${{ python-version }}
```

Comment:
@hawk-sergey-kurilo this will cause ai-scores to break if we don't specify the python version on the caller workflows inside the ai-scores repository. It's up to you to revert this change or contact @hawk-sarthak-garg to add it in ai-scores

Reply:
reverted this change